bin/plugin/commands/since.js

```diff
@@ -38,13 +43,32 @@ exports.handler = async ( opt ) => {
 		return;
 	}
 
-	const patterns = [
-		path.resolve( __dirname, '../../../**/*.php' ),
-		path.resolve( __dirname, '../../../**/*.js' ),
-	];
+	if ( opt.plugin && ! plugins.includes( opt.plugin ) ) {
+		log(
+			formats.error(
+				`The plugin "${ opt.plugin }" is not found in the plugins.json file.`
+			)
+		);
+		return;
+	}
+
+	const patterns = [];
+	const pluginRoot = path.resolve( __dirname, '../../../' );
+	const ignore = [ '**/node_modules', '**/vendor', '**/bin', '**/build' ];
+
+	if ( opt.plugin ) {
```

Comment:
Worth adding a comment to explain:
```suggestion
	/*
	 * For a standalone plugin, use the specific plugin directory.
	 * For Performance Lab, use the root directory and ignore the standalone plugin directories.
	 */
	if ( opt.plugin ) {
```